sourcegraph/cody
# Edit Input: Trigger doc and test commands in 1 click

## Description

This PR:
- Updates the Edit Input commands to action with a single click/keypress.
  - The range is driven from the "Range" section of the input.
  - Added "Select a Symbol..." quickpick that is opened via the "Range" section
    - Shows all symbols in the doc
    - Focuses on the nearest one to the users' cursor

Loom: https://www.loom.com/share/36a6c6701c1d40808389f69327923b03

## Test plan

1. Open Edit input
2. Select "Document Code" or "Generate Tests"
3. Expect the command actions in a single keypress, and works on the current range
4. Open Edit Input
5. Select "Range" and then "Select a Symbol"
6. Select a valid symbol
7. Select "Document Code" or "Generate Tests"
3. Expect the command actions in a single keypress, and works on the selected range
